File: BO_SPP/wwwroot/Scripts/Pages/AuditTrail.js
$(document).ready(function () {
    $('#TableData').DataTable({
        destroy: true,
        order: [[0, 'desc']],
        columns: [            
            { data: "Datetime" },
            { data: "Username" },            
            { data: "Menu" },
            { data: "Halaman" },           
            { data: "Action" },           
        ],
        fnInitComplete: function () {
            RefreshTable();
        }
    });     
});

function RefreshTable() {    

    $.ajax({
        url: VP + 'AuditTrail/GetAuditTrail',         
        type: 'POST',       
        success: function (Result) {
            if (Result.Error == false) {
                var table = $('#TableData').DataTable();
                table.clear().draw();
                if (Result.Message != null && Result.Message.length > 0) {
                    table.rows.add(Result.Message);
                    table.columns.adjust().draw();
                }
            } else {
                CustomNotif("error|Oops|" + Result.Message + "");
            }
        },
        error: function (xhr, status, error) {
            CustomNotif("error|Oops|" + error + "");
        }
    })

}

File: BO_SPP/Models/AuditTrail.cs
using System;

namespace BO_SPP.Models
{
    public class AuditTrail
    {
        public Guid? ID { get; set; }
        public string Username { get; set; }
        public string Menu { get; set; }
        public string Halaman { get; set; }
        public string Item { get; set; }
        public string Action { get; set; }
        public string Description { get; set; }
        public string Datetime { get; set; }

    }
}
